bundle-workflow/src/test_workflow/integ_test/local_test_cluster.py

```diff
@@ -22,12 +22,17 @@ class LocalTestCluster(TestCluster):
     Represents an on-box test cluster. This class downloads a bundle (from a BundleManifest) and runs it as a background process.
     """
 
-    def __init__(self, work_dir, bundle_manifest, security_enabled, s3_bucket_name):
+    def __init__(self, work_dir, component_name, additional_cluster_config, bundle_manifest, security_enabled,
+                 s3_bucket_name=None):
         self.manifest = bundle_manifest
         self.work_dir = os.path.join(work_dir, "local-test-cluster")
         os.makedirs(self.work_dir, exist_ok=True)
+        # Required by IM plugin as an additional config
+        os.makedirs("/tmp/tmp_dir", exist_ok=True)
```

Comment:
We should figure out a longer term solution for this. 
This should come from the test-manifest instead of being hard-coding it.

We could do something like: 
```
components:
  - name: index-management
    integ-test:
      test-configs:
        - with-security
        - without-security
      additional-cluster-configs:
        - "path.repo: /tmp/tmp_dir"
      shell-commands:
        - "mkdir /tmp/tmp_dir"
```

Reply:
The `path.repo` option feels wrong to me altogether, we shouldn't be over-engineering around some strange test. The test itself can create any paths it wants. What does it expect inside `path.repo`?

Reply:
Created the issue: https://github.com/opensearch-project/opensearch-build/issues/545